app/views/casa_cases/index.html.erb

```diff
@@ -93,4 +93,6 @@
     </div>
   </div>
 </div>
-<%= render 'other_duties' %>
+<% if FeatureFlagService.is_enabled?(FeatureFlagService::SHOW_OTHER_DUTIES_FLAG) %>
+  <%= render 'other_duties' %>
+<% end %>
```

Comment:
@whatnotery thanks for working on this! It sounds like from the ticket description that we still want to display existing "other duties", but just hide the "New Duty" button. Can we move the feature flag to the `other_duties` partial?

Reply:
Oh sure, easy fix!

Reply:
Thanks! So you could modify the system tests to test with and without the feature flag.